nautobot_ssot_ipfabric/diffsync/adapter_ipfabric.py

```diff
@@ -136,7 +136,7 @@ def load(self):
                         model=device.get("model") if device.get("model") else f"Default-{device.get('vendor')}",
                         vendor=device.get("vendor").capitalize(),
                         serial_number=serial_number,
-                        role=DEFAULT_DEVICE_ROLE,
+                        role=device.get("devType") if device.get("devType") else DEFAULT_DEVICE_ROLE,
```

Comment:
isn't this equivalent to:

```suggestion
                        role=device.get("devType", DEFAULT_DEVICE_ROLE) ,
```

Reply:
not if `device["devType"] == None`; I don't know if that's a concern though.

Reply:
I can't say for certain that it will never be `None` so I will leave this to handle it.

Reply:
It appears IPF `devType` will never be null but instead `unknown`.  Double checking this first.